### ISSUE TITLE
NPM/ES6: Export tabbedcontent as module

### ISSUE THREAD

**Issue by banditrider** [elboletaire/tabbedcontent#39]
Would be amazing if tabbedcontent was easy to import via `import` statement in ES6.

**Comment by elboletaire** [comment]
Yeah, in fact I wanna do the same with [password-strength-meter](https://github.com/elboletaire/password-strength-meter).

If you know how to do it, let me know or feel free to make a PR with the required changes 😃.
